config/bash/bash_setup_common.bash

```diff
@@ -141,3 +141,6 @@ androidpic_mv() {
 # 3/3/2014 to include rm_dropbox_conflictfiles.bash
 #export PATH=~/data/Dropbox/pg/Lateeye/bashapp:$PATH
 export PATH=~/link/github_repos/130s/compenv_ubuntu/util:$PATH
+
+# 6/9/2016 Workaround for tmux issue https://github.com/130s/compenv_ubuntu/issues/3
+export DISPLAY=:0
```

Comment:
Commenting in this line eliminated the issue https://github.com/kinu-garage/hut_10sqft/issues/804 on p16s...